charts/seed-controlplane/charts/etcd/templates/allow-from-apiserver-network-policy.yaml

```diff
@@ -6,6 +6,10 @@ metadata:
       Allows Ingress to etcd pods from the Shoot's Kubernetes API Server.
   name: allow-etcd
   namespace: {{ .Release.Namespace }}
+  labels:
+    garden.sapcloud.io/role: controlplane
+    app: etcd-statefulset
+    role: {{ .Values.role }}
```

Comment:
This should be removed, or?

Reply:
Which label should be removed? We need `role` to identify the etcd.